app/controllers/list_controller.rb

```diff
@@ -0,0 +1,43 @@
+class ListController < ApplicationController
+  def index
+    @lists = List.all
+  end
+
+  def edit
+    @list = List.find_by_id(params[:id])
```

Comment:
This will fail silently and with an arcane error message if the list is missing - I prefer `find` for controller methods so it 404s appropriately.